en/concept/mixing.md

```diff
@@ -74,16 +76,16 @@ For a multicopter things are a bit different: control 0 (roll) is connected to a
 * 1: RC pitch
 * 2: RC yaw
 * 3: RC throttle
-* 4: RC mode switch
-* 5: RC aux1
-* 6: RC aux2
-* 7: RC aux3
+* 4: RC mode switch (Passthrough of RC channel mapped by [RC_MAP_FLAPS](../advanced/parameter_reference.md#RC_MAP_FLAPS))
```

Comment:
FYI this makes it clear what this actually maps to!

Reply:
:+1: